code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -1301,6 +1301,13 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks)
 	ex_act(severity)
 		src.smash()
 
+
+	Cross(atom/movable/mover) //epic barfights
```

Comment:
```suggestion
	Crossed(atom/movable/mover) //epic barfights
```
`Cross` is to check if something is allowed to cross, `Crossed` is to do something when it does.